business/model/combined.go

```diff
@@ -0,0 +1,6 @@
+package model
+
+type CombinedResponse struct {
+	NetWorthData   NetWorthResponse
+	PendingJourney PendingJourneyResponse
```

Comment:
Response should match with the expected response

Reply:
done